tasks/list.yml

```
@@ -0,0 +1,15 @@
+# SPDX-License-Identifier: MIT
+---
+- name: List snapshots
+  ansible.builtin.script: "{{ __snapshot_cmd }}"
+  args:
+    executable: "{{ __snapshot_python }}"
+  register: snapshot_cmd
+
+- name: Print out response
+  debug:
+    var: snapshot_cmd.stdout
```

```suggestion
    var: snapshot_cmd.stdout
    verbosity: 2
```
like https://github.com/linux-system-roles/storage/blob/main/tasks/main-blivet.yml#L14
that way the user won't see this unless they use `ansible-playbook -vv` or the like

The user can always add their own `debug: var:` if they really want to print this out